modules/persistent-storage-csi-manila-share-access-rules.adoc

```diff
@@ -0,0 +1,108 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-manila.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="persistent-storage-csi-manila-share-access-rules_{context}"]
+= Customizing Manila share access rules
+
+By default, {product-title} creates Manila storage classes that provide access to all IPv4 clients. To limit client access, you can define custom storage classes that use specific client IP addresses or subnets by using the `nfs-ShareClient` parameter.
+
+[IMPORTANT]
+====
+When using custom storage classes with restricted access rules, ensure that:
+* The specified IP addresses or subnets include all {product-title} nodes that need to access the storage.
+* The Manila service in {rh-openstack} supports the share type specified in the storage class.
+* Network connectivity exists between the allowed clients and the Manila share servers.
+====
+
+.Prerequisites
+
+* {rh-openstack-first} is deployed with appropriate Manila share infrastructure.
+* Access to a cluster with administrator privileges.
+
+.Procedure
+
+. Create a YAML file for your custom storage class based on the following example:
++
+.Example custom storage class file
+[source,yaml]
+----
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: csi-manila-gold-restricted
+provisioner: manila.csi.openstack.org
+parameters:
+  type: gold
+  nfs-ShareClient: "10.0.0.0/24,192.168.1.100"
+  csi.storage.k8s.io/provisioner-secret-name: manila-csi-secret
+  csi.storage.k8s.io/provisioner-secret-namespace: openshift-manila-csi-driver
+  csi.storage.k8s.io/controller-expand-secret-name: manila-csi-secret
+  csi.storage.k8s.io/controller-expand-secret-namespace: openshift-manila-csi-driver
+  csi.storage.k8s.io/node-stage-secret-name: manila-csi-secret
+  csi.storage.k8s.io/node-stage-secret-namespace: openshift-manila-csi-driver
+  csi.storage.k8s.io/node-publish-secret-name: manila-csi-secret
+  csi.storage.k8s.io/node-publish-secret-namespace: openshift-manila-csi-driver
+allowVolumeExpansion: true
+----
++
+* `metadata.name`: Descriptive name for your custom storage class.
```

Comment:
Are we changing the style we describe config option, we usually use numbers (x) that we reference below.
example: 
https://docs.redhat.com/en/documentation/openshift_container_platform/4.19/html/storage/using-container-storage-interface-csi#persistent-storage-csi-group-snapshots-restore_persistent-storage-csi-group-snapshots

Reply:
Thought that we needed to change for the new tool, but that was premature. I've changed it back.